OpenRA.Game/Server/Server.cs

```diff
@@ -1243,17 +1342,32 @@ public void StartGame()
 				}
 
 				SyncLobbyInfo();
-				State = ServerState.GameStarted;
 
-				var disconnectData = new[] { (byte)OrderType.Disconnect };
+				if (LobbyInfo.GlobalSettings.UseNewNetcode)
+				{
+					serverGame = new ServerGame(LobbyInfo.GlobalSettings.Timestep);
+					foreach (var c in Conns)
+						serverGame.OrderBuffer.AddClient(c.PlayerIndex);
+
+					State = ServerState.GameStarted;
+					GameState = GameState.Loading;
+				}
+				else
+				{
+					State = ServerState.GameStarted;
+					GameState = GameState.Playing;
+				}
+
+				// TODO remove: this "pre-disconnecting" method adds unnecessary complexity just for the sake of the replay stream
+				/*var disconnectData = new[] { (byte)OrderType.Disconnect };
```

Comment:
Do we want to remove this code?

Reply:
I'm not sure yet, I haven't made replay and savegame testing a priority and I think that's where it was required. I like to think I actually fixed that one already.

Reply:
I'll remove it